devdox/app/services/repository.py

```
@@ -1,56 +1,61 @@
 from fastapi import Depends
-from tortoise.exceptions import IntegrityError
+from tortoise.exceptions import DoesNotExist, IntegrityError
 
+from app.exceptions import exception_constants
+from app.exceptions.base_exceptions import DevDoxAPIException
 from app.exceptions.local_exceptions import (
     BadRequest,
     ResourceNotFound,
 )
 from app.exceptions.exception_constants import (
+    GENERIC_ALREADY_EXIST,
```

Ruff (0.12.2)

`app.exceptions.exception_constants.GENERIC_ALREADY_EXIST` imported but unused

Remove unused import: `app.exceptions.exception_constants.GENERIC_ALREADY_EXIST`

(F401)